cinch/roles/jenkins_slave/templates/swarm.upstart.conf

```diff
@@ -0,0 +1,21 @@
+description "Jenkins slave swarm client"
```

Comment:
Good point about the syntax highlighting.  I prefer to have that working rather than following Ansible's conventions.